frontend/src/main/scala/bloop/Server.scala

```diff
@@ -51,6 +51,6 @@ object Server {
     import bloop.engine.State
     import bloop.engine.tasks.Tasks
     State.stateCache.allStates.foreach(s => Tasks.persist(s))
-    server.shutdown(true)
+    server.shutdown( /* exitVM = */ false)
```

Comment:
Why is this necessary?

Reply:
Because we shutdown the server at the end of the test, and we don't want to exit the JVM that sbt nicely forked for us.